sfcasts/strategy.md

```diff
@@ -1,119 +1,23 @@
 # Strategy Pattern
 
-Coming soon...
+The first pattern we'll talk about is the "strategy pattern". This is a *behavioral* pattern that helps you organize code into separate classes that can then interact with each other. Let's start with the *technical* definition: The strategy pattern defines a family of algorithms, encapsulates each one, and makes them *interchangeable*. It lets the algorithm vary independently from clients that use it. If that made sense to you, congrats! *You* should probably be teaching this course instead of me. So let's try that again. Here's *my* definition: The strategy pattern is a way to allow *part* of a class to be controlled from the *outside*.
 
-The first pattern we'll talk about is the strategy pattern, which is one of those
-behavioral patterns, the kind that help you organize code into separate classes that
-can then interact with each other. Let's start with the technical definition. The
-strategy pattern defines a family of algorithms, encapsulates each1 and makes them
-interchangeable. It lets the algorithm vary independently from clients that use it.
-Oh, if that made sense to you, congrats. You should probably be teaching this course
-instead of me. So let's try that again. Here's my definition. The strategy pattern is
-a way to allow part of a class to be controlled from the outside. Let's look at an
-imaginary example before we start coding. Suppose we have a payment service that does
-a bunch of stuff, including charging people via the credit card, but now we want to
-be able to use, sometimes use this exact same class except charge people via PayPal.
-Instead, how can we do that? The strategy pattern. Allow some new payment strategy
-interface object to be passed into payment service and just call that. Then create
-two classes that implement this credit card payment strategy and PayPal payment
-strategy.
+Let's talk about an *imaginary* example before we start coding. Suppose we have a payment service that does a bunch of stuff, including charging people via credit card. But *occasionally*, we'll want to use this exact same class to charge people via PayPal instead. How can we do that? The *strategy pattern*. We would allow a new payment strategy interface object to be passed into our payment service and just call *that*. Then we would create two classes that *implement* this: Credit card payment strategy and PayPal payment strategy. *You* now have control of which class you pass in. Yep! You just made part of the code *inside* payment service controllable by *outside* code. Cool! Now that we've got a quick preview, let's code a *real* example.
```

Comment:
For this part, you can keep the video blank (just freeze frame somewhere). I'll will, during my editing, put some code onto the screen and then I'll either have you or Leanna make it prettier ;).